GameEngine/Engine.cs

```diff
@@ -43,9 +43,27 @@ public static void Start(Func<IGameData> gameDataFactory, int loadFromSlot = 0)
                 {
                     // Otherwise if we're not starting a new game, show the game intro text
                     // Show the game introduction
+                    string finishedChange = gameData.FinalName();
```

Comment:
Maybe we can wrap all of this "Start new game" code into its own function

Reply:
Like a method?

Reply:
Done with that.